src/Hosting/Hosting/src/StaticWebAssets/StaticWebAssetsFileProvider.cs

```diff
@@ -158,4 +157,350 @@ public Stream CreateReadStream()
             }
         }
     }
+
```

Comment:
I've put the manifest side-by-side with the old manifest version because that one is going away after preview7 (for preview7 there is a way to fall back to the XML version to reduce the risk)